app/controllers/api/internal/sessions_controller.rb

```diff
@@ -20,13 +20,6 @@ def update
         render json: { live: live?, timeout: timeout }
       end
 
-      def destroy
```

Comment:
I think this is supposed to stay?

Reply:
> I've lost the thread on the iterations of this. Did a previous PR stop calling this already?
> 
> Also is there supposed to be a corresponding change to `routes.rb` removing this?

We shipped the API controllers to avoid conflicts with the 50%/50% state, but nothing is using them currently, so this should be safe.

But yes, I need to update `routes.rb`. Good catch 👍

Reply:
It was intentional to remove, because of the complications described in https://github.com/18F/identity-idp/pull/7966#issuecomment-1564773300 .

Reply:
Removed route in 6c658b3.